k-distribution/tests/regression-new/llvm-krun/Makefile

```diff
@@ -0,0 +1,34 @@
+DEF=test
+EXT=
```

Comment:
I'm not entirely sure why these tests are passing given that you didn't specify an extension. I'm not entirely sure you're actually running the test here, because ktest.mak chooses which files to run by looking for files with a certain extension. Please address.

Reply:
Sorted